editor/plugins/expression_evaluator_editor_plugin.cpp

```diff
@@ -77,44 +82,25 @@ void EditorExpressionEvaluator::_clear() {
 }
 
 void EditorExpressionEvaluator::_remote_object_selected(ObjectID p_id) {
-	editor_debugger->emit_signal(SNAME("remote_object_requested"), p_id);
+	// editor_debugger->emit_signal(SNAME("remote_object_requested"), p_id);
```

Comment:
Same here, though the original solution was already hacky. I wonder if there is better way.